README.md

```diff
@@ -1,454 +1,35 @@
 
-NVIDIA GPU Feature Discovery for Kubernetes is a software component that allows
-you to automatically generate labels for the set of GPUs available on a node.
-It leverages the [Node Feature
-Discovery](https://github.com/kubernetes-sigs/node-feature-discovery)
-to perform this labeling.
+## Migration
 
-## Beta Version
+All future development, bug fixes, and feature enhancements will take place in the new repository. Please update your references and bookmarks accordingly. 
 
-This tool should be considered beta until it reaches `v1.0.0`. As such, we may
-break the API before reaching `v1.0.0`, but we will setup a deprecation policy
-to ease the transition.
+## New Repository
 
-## Prerequisites
+The new repository, [k8s-device-plugin](https://github.com/NVIDIA/k8s-device-plugin), can be found at:
 
-The list of prerequisites for running the NVIDIA GPU Feature Discovery is
-described below:
-* nvidia-docker version > 2.0 (see how to [install](https://github.com/NVIDIA/nvidia-docker)
-and it's [prerequisites](https://github.com/nvidia/nvidia-docker/wiki/Installation-\(version-2.0\)#prerequisites))
-* docker configured with nvidia as the [default runtime](https://github.com/NVIDIA/nvidia-docker/wiki/Advanced-topics#default-runtime).
-* Kubernetes version >= 1.10
-* NVIDIA device plugin for Kubernetes (see how to [setup](https://github.com/NVIDIA/k8s-device-plugin))
-* NFD deployed on each node you want to label with the local source configured
-  * When deploying GPU feature discovery with helm (as described below) we provide a way to automatically deploy NFD for you
-  * To deploy NFD yourself, please see https://github.com/kubernetes-sigs/node-feature-discovery
+- https://github.com/NVIDIA/k8s-device-plugin
 
-## Quick Start
+## Contributions
 
-The following assumes you have at least one node in your cluster with GPUs and
-the standard NVIDIA [drivers](https://www.nvidia.com/Download/index.aspx) have
-already been installed on it.
+Contributions to this archived repository are no longer accepted. However, contributions to the new repository are welcome. Please refer to the contribution guidelines in the new repository for more information.
 
-### Node Feature Discovery (NFD)
+## Issues
 
-The first step is to make sure that [Node Feature Discovery](https://github.com/kubernetes-sigs/node-feature-discovery)
-is running on every node you want to label. NVIDIA GPU Feature Discovery use
-the `local` source so be sure to mount volumes. See
-https://github.com/kubernetes-sigs/node-feature-discovery for more details.
+Any issues or feature requests should be submitted to the issue tracker of the new repository. 
 
-You also need to configure the `Node Feature Discovery` to only expose vendor
-IDs in the PCI source. To do so, please refer to the Node Feature Discovery
-documentation.
+## License
```

Comment:
Is it required to add a license section?

Reply:
rewritten